WordPress/build.gradle

```diff
@@ -100,8 +100,9 @@ dependencies {
     compile 'com.android.support:percent:25.3.1'
     compile 'com.android.support:preference-v7:25.3.1'
 
-    compile 'com.google.android.gms:play-services-gcm:9.0.2'
-    compile 'com.google.android.gms:play-services-auth:9.0.2'
+    compile 'com.google.android.gms:play-services-gcm:10.0.1'
```

Comment:
Yes, using the latest version will likely prompt the users to update their play services (I prefer to let this to other apps).